client/checkout/platform-checkout/email-input-iframe.js

```diff
@@ -254,6 +255,9 @@ export const handlePlatformCheckoutEmailInput = async (
 	iframeWrapper.addEventListener( 'click', closeIframe );
 
 	const openIframe = ( email ) => {
+		if ( isLinkCheckoutActive( platformCheckoutEmailInput ) ) {
```

Comment:
This is my cheeky hack to interrupt the WooPay checkout modal, if Stripe Link has been selected. You can check the util function, but this is basically achieved by adding a new class to the email input and checking for its presence. 

FWIW the WooPay preloader still spins and this might be a little confusing, because ultimately (unless you change the payment gateway away from Link) nothing will happen. Let me know if you think this is bad!